Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/SpinBox.h

```diff
@@ -113,6 +193,12 @@ namespace AzQtComponents
         //! Qt::BlockingQueuedConnection as the parameters will only be valid for a short time
         void contextMenuAboutToShow(QMenu* menu, QAction* undoAction, QAction* redoAction);
 
+        // Like `QSpinBox::editingFinished'
+        void editingFinished();
```

Comment:
question: I'm not sure I 100% understand this sorry, is this virtual and needs an `override`? Or not, should we call it something else or do we rely on it hiding `editingFinished` in the base class if it's there?

Reply:
This isn't relevant to the new implementation, but yes, this relied on knowing the static (most derived) type when calling `editingFinished`